src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/waypoints_pending_resumption_handler.cc

```diff
@@ -36,99 +36,104 @@ namespace sdl_rpc_plugin {
 
 SDL_CREATE_LOG_VARIABLE("SdlRPCPlugin")
 
+using hmi_apis::FunctionID::Navigation_SubscribeWayPoints;
+
+uint32_t get_corr_id_from_message(const smart_objects::SmartObject& message) {
+  using namespace application_manager;
+  return message[strings::params][strings::correlation_id].asInt();
+}
+
 WayPointsPendingResumptionHandler::WayPointsPendingResumptionHandler(
     application_manager::ApplicationManager& application_manager)
     : PendingResumptionHandler(application_manager) {}
 
 smart_objects::SmartObjectSPtr
 WayPointsPendingResumptionHandler::CreateSubscriptionRequest() {
   SDL_LOG_AUTO_TRACE();
+
   auto subscribe_waypoints_msg =
       application_manager::MessageHelper::CreateMessageForHMI(
-          hmi_apis::FunctionID::Navigation_SubscribeWayPoints,
+          Navigation_SubscribeWayPoints,
           application_manager_.GetNextHMICorrelationID());
   (*subscribe_waypoints_msg)[application_manager::strings::params]
                             [application_manager::strings::message_type] =
                                 hmi_apis::messageType::request;
   return subscribe_waypoints_msg;
 }
 
+void WayPointsPendingResumptionHandler::SendHMIRequestForNotSubscribed(
```

Comment:
ForNotSubscribed makes me confused in this function name. 
Why not just: SendHMIRequest? or SendPendingHMIRequest

Reply:
Changed to SendPendingHMIRequest in fface93be6b07b30b9fe605959ace2d9b18b8f77